jekyll/_cci2/status-badges.md

```diff
@@ -37,6 +37,10 @@ To generate your own status badge, you will need to know, and substitute, the fo
 - `<LINK>` - The link you want the status badge to go to when clicked (example: the pipeline overview page)
 - Optional: an API token (to create badges for private projects)
 
+**Note:** Status Badges are currently only supported for bitbucket or github.
```

Comment:
We do support status badges for standalone orgs so docs should be good at the moment